source/guides/guides/test-retries.md

```diff
@@ -0,0 +1,193 @@
+
+{% img /img/guides/test-retries/cli-error-message.png %}
+
+During {% url "`cypress open`" command-line#cypress-open %} you will be able to see the number of attempts made in the {% url "Command Log" test-runner#Command-Log %} and expand each attempt for review and debugging if desired.
+
+{% video local /img/guides/test-retries/attempt-expand-collapse-time-travel.mp4 %}
+
+# Configure Test Retries
+
+## Global Configuration
+
+If you want to change the number of attempts being made for all tests run for both `cypress run` and `cypress open`, you can configure this in your {% url "configuration file" command-line#cypress-open-config-file-lt-config-file-gt %} (`cypress.json` by default) by defining the `retries` property and giving it the desired number of retries.
+
+```jsx
+{
+  "retries": 1
+}
+```
+
+### Disable Test Retries
+
+If you would like to disable test retries completely (the default configuration), you can use the global configuration to do so by setting `retries` to 0.
+
+```jsx
+{
+  "retries": 0
+}
+```
+
+### Configure specific retries for different modes
+
+If you want to define different retry attempts for `cypress run` versus `cypress open`, you can pass the `retries` option an object with the following options:
+
+- `runMode` allows you to define the number of test retries when running `cypress run`
+- `openMode` allows you to define the number of test retries when running `cypress open`
+
+```jsx
+{
+  "retries": {
+    // Configure retries for `cypress run`
+    // Default is 0
+    "runMode": 2,
+    // Configure retries for `cypress open`
+    // Default is 0
+    "openMode": 1
+  }
+}
+```
+
+## Custom Configurations
+
+### Individual Test(s)
+
+If you want a specific test to run a custom number of retries, you can set this by using the {% url "test's configuration" writing-and-organizing-tests#Test-Configuration %}.
+
+```jsx
+// Customize retries for an individual test
+describe('User sign-up and login', () => {
+  // `it` test block with no custom configuration
+  it('should redirect unauthenticated user to sign-in page', () => {
+    // ...
+  })
+
+  // `it` test block with custom configuration
+  it('allows user to login', {
+    retries: {
+      runMode: 2,
+      openMode: 1
+    }
+  }, () => {
+    // ...
+  })
+})
+```
+
+### Test Suite(s)
+
+If you want a suite of tests to re-run a custom number of times, you can do this by setting the suite's configuration.
+
+```jsx
+// Customizing retries for a suite of tests
+describe('User bank accounts', {
+  retries: {
+    runMode: 2,
+    openMode: 1,
+  }
+}, () => {
+  // Individual tests will be run normally
+  it('allows a user to view their transactions, () => {
+    // ...
+  }
+
+  it('allows a user to edit their transactions, () => {
+    // ...
+  }
+})
+```
+
+You can find more information about custom configurations here: {% url "Test Configuration" configuration#Test-Configuration %}
+
+# Screenshots
```

Comment:
I removed 'Videos' from this section. We only ever take a single video per `cypress run`. As far as I understand it - test retries would have zero effect on the actual video that is recorded.